mixin/thanos/alerts/rule.libsonnet

```diff
@@ -153,6 +153,33 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'ThanosRuleNoEvaluationForLast10Intervals',
+            annotations: {
+              message: 'Thanos Rule {{$labels.job}} have {{ $value | humanize }}% rule groups that did not evaluate for at least 10x of their expected interval.',
```

Comment:
```suggestion
              message: 'Thanos Rule {{$labels.job}} has {{ $value | humanize }}% rule groups that did not evaluate for at least 10x of their expected interval.',
```

Reply:
dude, you have `have` everywhere else (: Will fix there as well

Reply:
Well, then that's wrong. Also, I didn't write most of the other ones. Let's not start blaming here anyway :)